lms/templates/courseware/xqa_interface.html

```diff
@@ -28,7 +28,7 @@
             var username = $("#" + element_id + "_history_student_username").val();
             var location = $("#" + element_id + "_history_location").val();
             // xss-lint: disable=mako-invalid-js-filter
-            $("#" + element_id + "_history_text").load('/courses/' + "${six.text_type(course.id) | u}" +
+            $("#" + element_id + "_history_text").load('/courses/' + "${six.text_type(getattr(course,'id','')) | u}" +
```

Comment:
In what cases will a course ID be **null**? I don't why do we need to assign empty value?

Reply:
`course` object is not present on the instructor page while rendering ORA

Reply:
Then what will happen if the `id` is not found? What part of the modal will not be rendered?

Reply:
`Submission History Viewer` will not work properly. But don't know if it is needed on this page.